### DISCUSSION TITLE
Phigros x 节奏大师联动即将放出！

### DISCUSSION THREAD

**Discussion by 6EACzz** [share121/inter-knot#102] [其他]
![1723196848578](https://github.com/user-attachments/assets/b52ed83e-f941-407c-b62a-956cf10baffc)

Phigros将于2024年8月16日更新！本次更新将带来中国音乐节奏游戏「节奏大师」两首联动曲目。
#Phigros# #Pigeon Games# #鸽游#

**Comment by myt521** [comment]
真成节奏大师了
